browser/brave_account/brave_account_service_factory.cc

```diff
@@ -27,21 +41,31 @@ BraveAccountService* BraveAccountServiceFactory::GetFor(
       GetInstance()->GetServiceForContext(context, true));
 }
 
+// static
+BrowserContextKeyedServiceFactory::TestingFactory
+BraveAccountServiceFactory::GetDefaultFactory() {
+  return base::BindRepeating(&BuildBraveAccountService);
```

Comment:
nit: `base::BindOnce`. `TestingFactory` is `OnceCallback`.

Reply:
https://github.com/brave/brave-core/pull/31896/commits/71f39afa303e94b710051540032f8dc93c45ada5